Plan Chess From Outer Space/Assets/Scripts/BoardManager.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using UnityEngine;
+
+public class BoardManager : MonoBehaviour
+{
+    // --------------- Member variables and data --------------- //
+    // ChessBoard Settings
+    [Header("Chess Board Settings")] 
+    [SerializeField, Tooltip("Chess board width.")]
+    private int _boardWidth = 8;
```

Comment:
> Do we want to change the board size later?

Probably not, however having the ability to customize the board size is a nice to have feature.

> If so, make this public.

This is a bad idea, other classes should not be able to modify the board size on any case. We could add a public **getBoardSize()** method in case we need this data on another object.